docs/topics/selectors.rst

```diff
@@ -39,6 +39,7 @@ defines selectors to associate those styles with specific HTML elements.
 .. _XPath: https://www.w3.org/TR/xpath/all/
 .. _CSS: https://www.w3.org/TR/selectors
 .. _parsel: https://parsel.readthedocs.io/en/latest/
+.. _adding your own XPath extensions: https://parsel.readthedocs.io/en/latest/usage.html#parsel.xpathfuncs.set_xpathfunc
```

Comment:
I think you should be able to use a Sphinx reference for it instead of an explicitly written link.